Gemfile

```diff
@@ -70,6 +72,7 @@ gem "sanger_sequencing", "~> 0.0.1", path: "vendor/engines/sanger_sequencing"
 gem "secure_rooms", path: "vendor/engines/secure_rooms"
 gem "split_accounts", "~> 0.0.1", path: "vendor/engines/split_accounts"
 gem "synaccess_connect", "0.2.2", github: "tablexi/synaccess"
+gem "net-telnet"
```

Comment:
I remember this was an issue, but where does it get loaded? Was it a bundler thing? I feel like this would only get required if we used `Settings.relays.connect_module` as `NetBooter::Telnet`, which has been years since we used that option.